src/Arcus.Observability.Telemetry.Serilog.Enrichers/Extensions/LogEventPropertyValueExtensions.cs

```diff
@@ -86,6 +89,8 @@ public static bool TryGetAsDictionary(this IReadOnlyDictionary<string, LogEventP
         /// <remarks>The built-in <c>ToString</c> wraps the string with quotes</remarks>
         /// <param name="eventPropertyValues">Event property value to provide a string representation</param>
         /// <param name="propertyKey">Key of the property to return</param>
+        /// <exception cref="ArgumentNullException">Thrown when <paramref name="eventPropertyValues"/> is <c>null</c></exception>
```

Comment:
Added `<exception cref="ArgumentNullException">` and `<exception cref="NotSupportedException">` as they were missing.